File: good.pl
#!/usr/bin/perl

print("Hello world!\n");

use warnings;
use strict;
use JSON;
use Data::Dumper;

my $data_file = open(FH, '<', 'good_api.json') or die "good_api.json: $!";
print($data_file);
my $text = decode_json($data_file);

print Dumper($text);

close(FH);

File: good.py
#!/usr/bin/env python3

import json

def main():
    d = json.load(open('good_api.json'))
    print(d)

    entries = d['entries']
    print(entries[1]['author'])
    print(entries[0]['book'])

    for entry in entries:
        if entry['book'] == 'Dune':
            print(entry['author'])

if __name__ == "__main__":
   main()
